pkgs/development/interpreters/babashka/wrapped.nix

```diff
@@ -1,10 +1,20 @@
 { stdenv
+, lib
 , babashka-unwrapped
 , clojure
 , makeWrapper
 , rlwrap
 
 , jdkBabashka ? clojure.jdk
+
+  # rlwrap is a small utility to allow the editing of keyboard input, see
+  # https://book.babashka.org/#_repl
+  #
+  # NOTE In some cases, rlwrap prints some extra empty lines. That behavior can
+  # break some babashka scripts. For this reason, it is disabled by default. See:
+  # https://github.com/NixOS/nixpkgs/issues/246839
+  # https://github.com/NixOS/nixpkgs/pull/248207
+, withRlwrap ? false
 }:
 stdenv.mkDerivation (finalAttrs: {
```

Comment:
Can we change to `stdenvNoCC.mkDerivation` here? Because we don't need C compiler for anything in this derivation.

Reply:
Done, that makes sense. Honestly, I never used `stdenvNoCC.mkDerivation` before, and now I'm realizing that it makes more sense to use that one for almost all the derivations I wrote.